src/main/java/frc/robot/RobotContainer.java

```diff
@@ -85,6 +82,9 @@ private void configureButtonBindings() {
 
         sensitiveDriverControlCommand = new SensitiveDriverControl(driverStick);
 
+        JoystickButton turnToYaw = new JoystickButton(driverStick, XboxValue.XBOX_A_BUTTON.getPort());
+        turnToYaw.whenPressed(new TurnToOuterPortCommand(true, 3.0, 1.0));
```

Comment:
What are 3.0 and 1.0? Can we make these constants? If we feel this is the only time we will instantiate this class then maybe we can make these defaults in the construtor?

Reply:
So these are values which need to be tuned later so I'll make them into FluidConstants